xml/operations-compute-gpu_passthrough.xml

```diff
@@ -0,0 +1,272 @@
+<?xml version="1.0"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook51-profile.xsl"
+ type="text/xml"
+ title="Profiling step"?>
+<!DOCTYPE section [
+ <!ENTITY % entities SYSTEM "entity-decl.ent"> %entities;
+]>
+<section xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="gpu-passthrough">
+  <title>GPU passthrough</title>
+  <para>
+    &productname; GPU passthrough functionality provides the nova instance direct access to the GPU device for a
+    better performance.
```

Comment:
"for a better performance" should be "for better performance"

Reply:
updated in the latest commit